docs/hacking_howto.rst

```diff
@@ -90,6 +90,8 @@ These are required for the minimum installation:
 * libxslt and headers
 * libjpeg and headers
 * zlib and headers
+* libssl library
+* libmysqlclient library
```

Comment:
This line is covered by the MariaDB 5.5 server and client headers line.

Why do we need libssl? Also, it's probably not that you need libssl, but rather that you need the headers.